dynamicforms/mixins/conditional_visibility.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import annotations
+
+from enum import IntEnum
+from typing import Tuple, Union, Iterable
+
+
+class DependantVisibilityMixin(object):
+    def __init__(self, *args, conditional_visibility: Statement = None, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.conditional_visibility = conditional_visibility
+
+    def as_component_def(self) -> dict:
+        try:
+            res = super().as_component_def()  # noqa
+        except AttributeError:
+            res = dict()
+        res.update(dict(conditional_visibility=self.conditional_visibility.to_value()))
+        return res
+
+
+class Operators(IntEnum):
+    # Logic Operators
+    NOT = 0
+    OR = 1
+    AND = 2
+    XOR = 3
+    NAND = 4
+    NOR = 5
+
+    # Comparators
+    EQUALS = -1
+    NOT_EQUALS = -2
+    GT = -3
+    LT = -4
+    GE = -5
+    LE = -6
+    INCLUDED = -7
```

Comment:
IN? V najslabšem primeru INCLUDES. Nikakor pa ne INCLUDED
manjka tudi NOT IN. Ali je pač NOT IN == NOT(IN)?